src/utils/acorn-static-class-features.js

```diff
@@ -0,0 +1,81 @@
+"use strict"
+
+const acorn = require("acorn")
+const tt = acorn.tokTypes
+
+function maybeParseFieldValue(field) {
+  if (this.eat(tt.eq)) {
+    const oldInFieldValue = this._inStaticFieldValue
+    this._inStaticFieldValue = true
+    field.value = this.parseExpression()
+    this._inStaticFieldValue = oldInFieldValue
+  } else field.value = null
+}
+
+const privateClassElements = require("acorn-private-class-elements")
+
+module.exports = function(Parser) {
+  const ExtendedParser = privateClassElements(Parser)
```

Comment:
I'm confused why this plugin was added?

Is this just a patched version of an existing plugin?
Or did it never exist before?

Reply:
Originally I must admit I had no idea why this PR was working. It seems like this fix is actually due to the latest commit on master at https://github.com/acornjs/acorn-static-class-features/commit/e7295fc9ea64d46f166f53c757884dd9b0d1e177 which isn't currently released.